### PR TITLE
Kateryna

### PR DESCRIPTION
**Pull request by kate-ka**
Added new file

### CODE REVIEW

**Comment by kate-ka** [comment]
Added folder kateryna

File: classes/composition_example.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    def assign_teaching(self, course):
+        self.courses_taught.append(course)
+
+
+class Person:
+    def __init__(self, name, surname, number, learner=None, teacher=None):
+        self.name = name
+        self.surname = surname
+        self.number = number
+        #add behaviours as attrubutes insted of inheritance
```
Comment (UkrainianEngineer) [inline classes/composition_example.py]: Try to use full sentences.

Example:
`# Add behaviors as attributes instead if inheritance.`

**Comment by UkrainianEngineer** [comment]
@kate-ka , please assign any assignee

**Comment by UkrainianEngineer** [comment]
Reopened pull request.
We should have answered all the comments.

**Comment by kate-ka** [comment]
@pivancy  Done

**Comment by UkrainianEngineer** [comment]
I can't see suggested changes.

**Comment by kate-ka** [comment]
@pivancy  What changes do you mean?

File: classes/composition_example.py (hunk 1 above)
Comment (UkrainianEngineer) [reply classes/composition_example.py]: @kate-ka , I'm asking about changes here.

Comment (kate-ka) [reply classes/composition_example.py]: Done

**Comment by UkrainianEngineer** [comment]
@kate-ka , this pull request may be closed or merged into `release-2` branch.

P.S. don't forget to replace `release` branch with `release-2` if you decide to merge this.

**Comment by kate-ka** [comment]
As there are no changes I'm closing it.

**Pull request closed without merging**